### ISSUE TITLE
Bank verification needs manual intervention

### ISSUE THREAD

**Issue by tremchez** [allegro/allegro-api#10136]
Hi, our sandbox account needs manual verification for the bank account. Can you help?

**Comment by PrzemyslawLukanowski** [comment]
Yes, sure. Can you send me the login or email address associated with your account?

**Comment by tremchez** [comment]
> Yes, sure. Can you send me the login or email address associated with your account?

Sure, it's "svcgroupcztest" thanks.

**Comment by PrzemyslawLukanowski** [comment]
Thank you, I've activated your account.
